multi-deposit-instructions.md

```diff
@@ -170,9 +170,8 @@ a audio or video presentation contained in the dataset:
 * The data provided in `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION` are stored for further processing
   in the `deposit.properties` file.
 * `SF_PLAY_MODE` specifies how the video's are played in Springfield. The value must either be
-  `continuous` or `menu`. If no value is specified for a dataset with video/audio, the default will
-  be set to `continuous`. This value is only allowed if `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION`
-  are provided as well.
+  `continuous` or `menu`. This value is only allowed if `SF_DOMAIN`, `SF_USER` and `SF_COLLECTION`
+  are provided as well. When `menu` is chosen, **every A/V file** must have `FILE_TITLE` defined as well.
```

Comment:
```suggestion
  are provided as well. If `menu` is chosen, **every A/V file** must have `FILE_TITLE` defined as well.
```
Sorry for nitpicking, but I think in English this distinction is more strict than the corresponding one between "wanneer" en "als" in Dutch. See: https://dictionary.cambridge.org/us/grammar/british-grammar/if-or-when